service/src/main/java/skills/services/admin/InviteOnlyProjectService.groovy

```diff
@@ -168,9 +175,13 @@ class InviteOnlyProjectService {
         } else if(!projectAccessToken.isValid()) {
             response.valid = false
             response.message = "Project Invite has expired"
+        } else if(validateInviteEmail && userInfoService.getCurrentUser()?.email != projectAccessToken.recipientEmail) {
```

Comment:
since invite emails are user entered then email comparison must be case insensitive